qt/aqt/__init__.py

```diff
@@ -455,8 +463,9 @@ def msgHandler(category: Any, ctx: Any, msg: Any) -> None:
     if driver == VideoDriver.OpenGL:
         # Leaving QT_OPENGL unset appears to sometimes produce different results
         # to explicitly setting it to 'auto'; the former seems to be more compatible.
-        pass
-    else:
+        if qtmajor > 5:
+            QQuickWindow.setGraphicsApi(QSGRendererInterface.GraphicsApi.OpenGL)
```

Comment:
While QQuickWindow is used here, this should affect QWidgets too. You can confirm that by checking `mw.windowHandle().surfaceType()`.

Reply:
Setting `QSG_RHI_BACKEND` here didn't work too BTW. It only works if I set it before all Qt imports.